src/test/java/com/devonfw/sample/archunit/ArchitectureTest.java

```diff
@@ -16,15 +16,14 @@ public class ArchitectureTest {
   @ArchTest
   private static final ArchRule shouldOnlyAccessValidLayers = //
       layeredArchitecture().consideringAllDependencies() //
-          .layer("common").definedBy("..common..") //
-          .layer("logic").definedBy("..logic..") //
-          .layer("dataaccess").definedBy("..dataaccess..") //
-          .layer("service").definedBy("..service..") //
-          .layer("batch").definedBy("..batch..") //
-          .layer("ui").definedBy("..ui..") //
-          // TODO
-          .because("Dependency of technical layers violates architecture rules.");
+          .layer("common").definedBy("com.devonfw.sample.archunit.common..") //
+          .layer("logic").definedBy("com.devonfw.sample.archunit.logic..") //
+          .layer("dataaccess").definedBy("com.devonfw.sample.archunit.dataaccess..") //
+          .layer("service").definedBy("com.devonfw.sample.archunit.service..") //
+          .layer("client").definedBy("com.devonfw.sample.archunit.client..")
```

Comment:
@vlad961 the package root and especially the component should not be encoded here:
* We might have multiple components in the same application and each could have any up to all out of this layers
* As we want to provide a "starting point" and "template" for projects using devonfw and arch-unit we should not tie this to `com.devonfw.sample.archunit` as otherwise every project would have to adopt this to their own root package what is however not neccessary.

Reply:
I have changed each layer definition by exchanging the prefix: "com.devonfw.sample.archunit" with a second "." (dot).